connectors/v2/youtube.js

```diff
@@ -14,10 +14,28 @@ chrome.storage.local.get('Connectors', function(data) {
 	}
 });
 
-Connector.playerSelector = '#page';
+Connector.videoSelector = '#player-api .html5-main-video';
+
+{
+	$(Connector.videoSelector).on('timeupdate', () => {
+		if (Connector.isStateChangeAllowed()) {
```

Comment:
I'm just wondering why did you put event listener setup into curly braces? Is there a certain reason for that?